### DISCUSSION TITLE
How do I purge the cache programatically?

### DISCUSSION THREAD

**Discussion by mastoj** [vercel/community#3573] [Help]
### Summary

Is there a way to programatically purge the cache completely? The functions `revalidatePath` and `revalidateTag` isn't sufficient if I want to clear the whole cache. Having to do so manually in the console is hard to automate, and there might be other scenarios than deploy that should trigger a purge.

### Steps to Reproduce

Not much to reproduce, just need a pointer/answer regarding my question.

**Comment by amyegan** [comment]
Hey @mastoj. Redeploying is the recommended way to clear the cache - https://github.com/vercel/community/discussions/457#discussioncomment-19810

You could set up a GitHub Action or Cron Job to redeploy on a fixed schedule if that's something your site needs. Setup would be similar to the steps in this guide: https://vercel.com/guides/how-can-i-use-github-actions-with-vercel
